### DISCUSSION TITLE
Lab: Safe HTML Filter 2 - Returns False Error

### DISCUSSION THREAD

**Discussion by CuriousMindz1981** [paj28/xssy#21] [Q&A] [answered]
Hey all,

Firstly, great XSS playground, I was looking for exactly this in prep for my exam.

However, The Lab: Safe HTML Filter 2 I have popped the XSS using my payload which of course captures the cookie flag as instructed.

However, when using the submit function for this, it's always returning false, which if that was the case, I wouldn't have solved the challenge.

**Comment by CuriousMindz1981** [comment]
![Evidence 2](https://github.com/paj28/xssy/assets/23474549/050a4237-c332-47f6-9e16-7a0c59694196)
![Evidence](https://github.com/paj28/xssy/assets/23474549/ab70f374-60fa-445b-aac0-e9aa5d07313b)

**Comment by paj28** [comment, marked as answer]
Glad you are enjoying the labs. The headless browser is Chrome, so you need an exploit that works in Chrome. In the short term I'll add some text that clarifies this. In the long term, I intend to have headless Firefox as well.

**Reply by CuriousMindz1981** [reply]
Thanks for the speedy reply!

interestingly, one exercise in the easy section did the same too, but allowed the others with Firefox.

Hopefully you continue making these labs as they’re very good, especially when prepping for an exam :)
